### DISCUSSION TITLE
How to using 'nbe' scripts

### DISCUSSION THREAD

**Discussion by twquan1** [ConnectEverything/nats-by-example#117] [Q&A]
I had tried to download nbe file and paste at both "C:\nats-by-example-0.1.3" and "..\cmd\nbe" folder, then run the nbe script by this directory:
![image](https://github.com/ConnectEverything/nats-by-example/assets/56615342/78a79bac-54fc-478a-a410-5332bbe9cf40)
However I would receive this message, what do I did wrong here? 

Sorry I am beginner here.

Edits: I had change the nbe file format into ".exe", and the cmd able to run the command alrdy but I received new error message like this:
![image](https://github.com/ConnectEverything/nats-by-example/assets/56615342/91aa14e4-4579-4461-9026-3939d5a3f021)
I had installed docker desktop but this error are popping up instead, what configuration I still needed to edit to able to run this nbe cmd?

**Comment by bruth** [comment]
Hi @twquan1. Using `nbe run <example-name>` is the right invocation, but I have not tested this on Windows officially, so it is likely there some are some incompatible interactions happening. For example, your screenshot suggests to me the way I am creating the image seems incompatible with how Windows escapes path.

I am going to change this to a bug and see if I can setup a test environment for Windows to isolate and fix these issues.

**Reply by debbyglance** [reply]
Hello, any update on this?  I am running into the same problem

**Comment by Soundman32** [comment]
Same here.  I downloaded the Windows version of nbe (and renamed to nbe.exe), now I'm getting: "invalid reference format"

> D:\source\nats-by-example>nbe run messaging/pub-sub/cli
[+] Building 0.0s (0/0)                                                                                  docker:default
ERROR: invalid tag "nbe\\messaging\\pub-sub\\cli:590042e3": invalid reference format
build image: exit status 1

**Comment by scottf** [comment]
See #118
